.ci/setup_env_centos.sh

```diff
@@ -63,6 +63,7 @@ declare -A packages=( \
 	[qemu_dependencies]="libcap-devel libcap-ng-devel libattr-devel libcap-ng-devel librbd1-devel flex libfdt-devel libpmem-devel ninja-build" \
 	[kernel_dependencies]="elfutils-libelf-devel flex pkgconfig patch" \
 	[crio_dependencies]="glibc-static libseccomp-devel libassuan-devel libgpg-error-devel util-linux libselinux-devel" \
+	[gperf_dependencies]="gcc-c++" \
```

Comment:
Debian and Ubuntu get this by default as part of the `build-essential` package, but for consistency, we should probably add the C++ compiler for RHEL, SLES and OpenSUSE too.

Reply:
Thank you for your review. Sorry, I missed it. 
I fixed it and also added a commit that is a minor fix about blank.